src/workers/hermes_strategies/upload_strategy.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+export default class HermesUploadStrategy {
```

Comment:
Imagine scenario: I want to check fee and then decide if I want to bundle.
Idea 1: inject dataModelEngine to the strategy
Idea 2: make method desiredComponents or similar with component names and inject them from the builder

Reply:
Wowowow... Does not look beautiful, but maybe it is needed...
@spherefoundry ???